baseplate/events/queue.py

```diff
@@ -21,8 +21,28 @@
 import time
 import uuid
 
+from enum import Enum
+
 from . import MAX_EVENT_SIZE, MAX_QUEUE_SIZE
 from baseplate.message_queue import MessageQueue, TimedOutError
+from baseplate._utils import warn_deprecated
+
+
+class FieldKind(Enum):
+    """Enum of field kinds."""
+    NORMAL = None
```

Comment:
also something's going funky when the docs generate: it says `None` for each of them